### DISCUSSION TITLE
running multiple vector configs at same time in parallel

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#15217] [Q&A] [answered]
Hi. is there any way to run multiple vector configs like `vector --config ./` 
my problem is if one of these vector configs have an syntax error , none of them works or something .
is there any way to run all of them except one who has a syntax error.

**Comment by jszwedko** [comment, marked as answer]
Hi @LinTechSo ! Vector is currently all or nothing. Are the configurations completely separate? Or do the depend on each other? If separate, you could run separate Vector processes with each separate config file.
